plume-common/src/activity_pub/mod.rs

```diff
@@ -129,38 +130,38 @@ where
         .sign(sender)
         .expect("activity_pub::broadcast: signature error");
 
+    let mut rt = tokio::runtime::current_thread::Runtime::new().expect("new rt");
```

Comment:
if this fails, we panic with "new rt".??

Reply:
Hum… yes, I could probably change that. To be honest I copied this line from reqwest examples without really thinking about it :sweat_smile: